src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -117,10 +120,29 @@ public static void addDocumentToBundle(FooterService footerService, SscsCaseData
     }
 
     public static DocumentType getPostHearingReviewDocumentType(PostHearing postHearing) {
-        if (SetAsideActions.REFUSE.equals(postHearing.getSetAside().getAction())) {
-            return DocumentType.SET_ASIDE_REFUSED;
-        } else if (CorrectionActions.REFUSE.equals(postHearing.getCorrection().getAction())) {
-            return DocumentType.CORRECTION_REFUSED;
+        if (nonNull(postHearing.getReviewType())) {
```

Comment:
The nonNull check isn't necessary if you throw an exception on the default case.

Reply:
switch statements cant handle null values